container-toolkit/cdi-support.md

```diff
@@ -39,8 +39,12 @@ CDI also improves the compatibility of the NVIDIA container stack with certain f
 
 ### Procedure
 
-Two typical locations for CDI specifications are `/etc/cdi/` and `/var/run/cdi`.
-However, the path to create and use can depend on the container engine that you use.
+Two common locations for CDI specifications are `/etc/cdi/` and `/var/run/cdi/`.
+NVIDIA recommends using the `/etc/cdi/` directory for user-applied changes, such as performing the following steps.
```

Comment:
I don't know whether I want to make this recommendation at this stage. I can see us moving to using `/var/run/cdi` by default as soon as we have tooling to generate these specs at boot.